Makefile

```
@@ -128,7 +130,7 @@ run: manifests build
 
 .PHONY: docker-build
 docker-build:
-	IMG=$(IMG) docker build -t ${IMG} --build-arg TARGETOS=${TARGETOS} --build-arg TARGETARCH=${TARGETARCH} .
+	IMG=$(IMG) docker build -t ${IMG} --build-arg TARGET_OS=${TARGET_OS} --build-arg TARGET_ARCH=${TARGET_ARCH} --build-arg VERSION=${VERSION} --build-arg BUILD_TIME="$(shell date +'%Y-%m-%d %H:%M:%S')" .
```

`BUILD_TIME` isn't used.
```suggestion
	IMG=$(IMG) docker build -t ${IMG} --build-arg TARGET_OS=${TARGET_OS} --build-arg TARGET_ARCH=${TARGET_ARCH} --build-arg VERSION=${VERSION} .
```